Extensions/dist/extensions.js

```diff
@@ -157,3 +167,11 @@ function writeListFile(extensions) {
   });
   writePageFile('list.json', JSON.stringify(list));
 }
+
+if (require.main === module) {
```

Comment:
But since this is getting hooked into the `gulpfile.js` as `gulp build:extensions` do we even need to be able to execute this file manually?  Why wouldn't we just call `gulp build:extensions`?